### PR TITLE
Feature/kah 30

### PR DESCRIPTION
**Pull request by ChBernat**

### CODE REVIEW

File: server/api/index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+const express = require('express');
```
Comment (arispen) [inline server/api/index.js]: why this file is in `server/api` ? it is super confusing

File: server/app.js [hunk 2]
```diff
@@ -1,126 +1,8 @@
...
+const bootstrap = require("./helpers/bootstrap").bootstrap;
 
-module.exports = {
-    server: server.listen(port, () => {
+server.listen(port, () => {
         winston.log('info', `Running on localhost:${port}`);
         bootstrap(app);
-    })
-};
+    });
```
Comment (arispen) [inline server/app.js]: fix formatting plz

**Comment by arispen** [comment]
> Tests are no closing after running. The process is waiting for something but there is no info for what.

it works for me

**Comment by ChBernat** [comment]
@theflexbird Did you try `npm t`? I can't really recreate the issue.

**Pull request merged**